modules/apps/document-library/document-library-api/src/main/resources/com/liferay/document/library/configuration/packageinfo

```diff
@@ -1 +1 @@
-version 1.3.1
\ No newline at end of file
+version 1.4.0
```

Comment:
@adolfopa this is ok with the release or we most stop and do not forward?

Reply:
Mmm, it is not backwards compatible so we need to wait :-(

Reply:
Nevermind. We have until tomorrow to send these kinds of changes, so if everything is green today we should be safe.